### PR TITLE
Update F# Interactive Dependency Manager Plugins README.md

### PR DESCRIPTION
**Pull request by smoothdeveloper**
Links to microsoft.com documentation for FSI references and command line parameters.

Precise how they are deployed.

### CODE REVIEW

File: src/FSharp.DependencyManager.Nuget/README.md [hunk 1]
```diff
@@ -49,4 +55,6 @@ X,Y
 
 for r in MyCsv.GetSample().Rows do
   printfn "%i = %s" r.X r.Y
-```
\ No newline at end of file
+```
+[^1]: [Referencing packages in F# Interactive](https://learn.microsoft.com/en-us/dotnet/fsharp/tools/fsharp-interactive/#referencing-packages-in-f-interactive)
+[^2]: [F# Interactive options](https://learn.microsoft.com/en-us/dotnet/fsharp/language-reference/fsharp-interactive-options)
```
Comment (psfinaki) [inline src/FSharp.DependencyManager.Nuget/README.md:60]:
```suggestion
[^2]: [F# Interactive options](https://learn.microsoft.com/dotnet/fsharp/language-reference/fsharp-interactive-options)
```

File: src/FSharp.DependencyManager.Nuget/README.md [hunk 2]
```diff
@@ -49,4 +55,6 @@ X,Y
 
 for r in MyCsv.GetSample().Rows do
   printfn "%i = %s" r.X r.Y
-```
\ No newline at end of file
+```
+[^1]: [Referencing packages in F# Interactive](https://learn.microsoft.com/en-us/dotnet/fsharp/tools/fsharp-interactive/#referencing-packages-in-f-interactive)
```
Comment (psfinaki) [inline src/FSharp.DependencyManager.Nuget/README.md:59]:
```suggestion
[^1]: [Referencing packages in F# Interactive](https://learn.microsoft.com/dotnet/fsharp/tools/fsharp-interactive/#referencing-packages-in-f-interactive)
```

File: src/FSharp.DependencyManager.Nuget/README.md [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
 
-The current implementation follows pattern that can be deducted by refering to [implementation in DependencyProvider.fs](https://github.com/dotnet/fsharp/blob/b9687a58cee795a94eb88cf84e309767cc25f6cb/src/Compiler/DependencyManager/DependencyProvider.fs#L145-L322); due to the system working without having a staticically linked dependency, it uses a late binding approach leveraging runtime reflection to identify if an extension conforms to patterns understood by the specifics of the compiler implementation.
+More about F# Interactive References
 
+The current implementation follows pattern that can be deducted by refering to [implementation in DependencyProvider.fs](https://github.com/dotnet/fsharp/blob/b9687a58cee795a94eb88cf84e309767cc25f6cb/src/Compiler/DependencyManager/DependencyProvider.fs#L145-L322); due to the system working without having a staticically linked dependency, it uses a late binding approach leveraging runtime reflection to identify if an extension conforms to patterns understood by the specifics of the compiler implementation.
```
Comment (psfinaki) [inline src/FSharp.DependencyManager.Nuget/README.md:21]:
```suggestion
The current implementation follows pattern that can be deducted by referring to [implementation in DependencyProvider.fs](https://github.com/dotnet/fsharp/blob/b9687a58cee795a94eb88cf84e309767cc25f6cb/src/Compiler/DependencyManager/DependencyProvider.fs#L145-L322); due to the system working without having a statically linked dependency, it uses a late binding approach leveraging runtime reflection to identify if an extension conforms to patterns understood by the specifics of the compiler implementation.
```

File: src/FSharp.DependencyManager.Nuget/README.md [hunk 4]
```diff
@@ -1,17 +1,24 @@
...
+For F# Interactive to load the extensions[^1], they either need to:
+* be deployed next to `fsi.dll` of dotnet SDK (or next `FsiAnyCPU.exe` for .NET Framework)
+* refered to via `--compilertool:<extensionsfolderpath>`[^2] argument
+
+The same goes for the tooling hosting [F# Compiler Service](https://fsharp.github.io/fsharp-compiler-docs/fcs/).
+
 The initial [RFC](https://github.com/fsharp/fslang-design/blob/main/tooling/FST-1027-fsi-references.md) for this feature overviews how it is designed.
 
-The current implementation follows pattern that can be deducted by refering to [implementation in DependencyProvider.fs](https://github.com/dotnet/fsharp/blob/b9687a58cee795a94eb88cf84e309767cc25f6cb/src/Compiler/DependencyManager/DependencyProvider.fs#L145-L322); due to the system working without having a staticically linked dependency, it uses a late binding approach leveraging runtime reflection to identify if an extension conforms to patterns understood by the specifics of the compiler implementation.
+More about F# Interactive References
```
Comment (psfinaki) [inline src/FSharp.DependencyManager.Nuget/README.md:19]:
```suggestion
**More about F# Interactive References**
```

**Review by psfinaki — APPROVED** [review]
Some accessibility remarks. Thanks!

File: src/FSharp.DependencyManager.Nuget/README.md (hunk 3 above)
Comment (smoothdeveloper) [reply src/FSharp.DependencyManager.Nuget/README.md:21]: oh my bad :D 

I'm thinking the nuget one (which is kind of template for now) should provide named SRTP constraints in a separate file that extensions can easily pull (via paket git dependencies or such), to make it statically checked despite it is not linked.

**Review by T-Gro — APPROVED** [review]

**Pull request merged**